app/config/paths.js

```diff
@@ -6,7 +6,11 @@ const isDev = require('electron-is-dev');
 
 const cfgFile = '.hyper.js';
 const defaultCfgFile = 'config-default.js';
-const homeDir = homedir();
+let homeDir = homedir();
+
+if (process.platform === 'win32') {
+  homeDir = resolve(homeDir, 'AppData\\Roaming\\Hyper');
```

Comment:
Any reason to not use `app.getPath('userData')`? That actually solves this issue, but also adds support for XDG_CONFIG_HOME and such.